pom.xml

```diff
@@ -5,36 +5,44 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.4.6</version>
+		<version>2.6.1</version>
 		<relativePath/>
 	</parent>
 	<groupId>ch.admin.bag.covidcertificate</groupId>
 	<artifactId>cc-management-service</artifactId>
-	<version>3.0.2</version>
+	<version>3.0.3</version>
 	<name>cc-management-service</name>
 	<description>Service for generating Covid Certificates</description>
 
 	<properties>
 		<java.version>11</java.version>
 
-		<spring-cloud.version>2020.0.3</spring-cloud.version>
-		<logstash.version>6.6</logstash.version>
-		<janino.version>3.1.4</janino.version>
-		<cc-backend-logging.version>1.0.0</cc-backend-logging.version>
-		<guava.version>30.1.1-jre</guava.version>
-        <openpdf.version>1.3.26</openpdf.version>
-		<springdoc.version>1.5.8</springdoc.version>
-		<jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
+		<cc-backend-logging.version>1.0.1</cc-backend-logging.version>
+		<com.github.tomakehurst.version>2.16.0</com.github.tomakehurst.version>
+		<com.openhtmltopdf.version>1.0.10</com.openhtmltopdf.version>
+		<com.squareup.okhttp3.version>4.9.3</com.squareup.okhttp3.version>
 		<dgc-java.version>1.0.1</dgc-java.version>
-		<spring-cloud-starter-sleuth.version>3.0.3</spring-cloud-starter-sleuth.version>
+		<google-zxing.version>3.4.1</google-zxing.version>
+		<guava.version>31.0.1-jre</guava.version>
 		<httpclient.version>4.5.13</httpclient.version>
+		<jacoco-maven-plugin.version>0.8.7</jacoco-maven-plugin.version>
+		<janino.version>3.1.6</janino.version>
+		<jfixture-mockito.version>2.7.2</jfixture-mockito.version>
 		<jjwt.version>0.11.2</jjwt.version>
-		<google-zxing.version>3.3.0</google-zxing.version>
-		<opencsv.version>5.4</opencsv.version>
 		<juniversalchardet.version>2.4.0</juniversalchardet.version>
-
+		<logstash.version>6.6</logstash.version>
+		<net.therore.logback.version>1.0.0</net.therore.logback.version>
+		<opencsv.version>5.5.2</opencsv.version>
+		<openpdf.version>1.3.26</openpdf.version>
+		<org.jsoup.jsoup.version>1.14.3</org.jsoup.jsoup.version>
+		<org.thymeleaf.thymeleaf.version>3.0.11.RELEASE</org.thymeleaf.thymeleaf.version>
+		<spring-cloud-starter-sleuth.version>3.0.4</spring-cloud-starter-sleuth.version>
+		<spring-cloud.version>2020.0.4</spring-cloud.version>
+		<springdoc.version>1.5.13</springdoc.version>
 		<maven.javadoc.skip>true</maven.javadoc.skip>
 
+		<log4j2.version>2.15.0</log4j2.version>
```

Comment:
version 2.16.0 should be the choice. The jndi-lookup code has been completly removed there. 2.15.0 seems to be produced in a hurry.